python/nammu/utils/__init__.py

```diff
@@ -150,6 +164,39 @@ def fix_old_default_project_yaml(yaml):
     return yaml
 
 
+def patch_styles(yaml):
+    '''
+    Following the release of 1.0 any user with a config generated by version
+    0.8 will crash without this method to restructure the style parts of the
+    config file.
+    '''
+    if 'console_style' in yaml.keys():
+        yaml['console_style'] = {'fontsize': {'default': 11, 'user': 11},
+                                 'background_color': {'default': 'Gray',
+                                                      'user': 'Gray'},
+                                 'font_color': {'default': 'Black',
+                                                'user': 'Black'}}
+
+    if 'edit_area_style' in yaml.keys():
+        yaml['edit_area_style'] = {'fontsize': {'default': 14, 'user': 14}}
+
+    return yaml
+
+
+def patch_server_settings(yaml):
+    '''
+    Following the change in server address we need to update user's config
+    files without overwriting their other settings.
+    '''
+    old_url = 'http://oracc.museum.upenn.edu'
+    new_url = 'http://build-oracc.museum.upenn.edu'
+    if 'servers' in yaml.keys():
```

Comment:
Maybe add:
if 'servers' in yaml.keys() and 'upenn' in yaml['servers']

Reply:
Resolved in dcdd71f571.